main/lsp/LSPConfiguration.cc

```diff
@@ -51,6 +50,7 @@ LSPClientConfiguration::LSPClientConfiguration(const InitializeParams &params) {
         } else {
             rootUri = *rootUriString;
         }
+        workspaceRootPath = string(absl::StripPrefix(rootUri, "file://"));
```

Comment:
At Stripe, Sorbet runs on a server, and VS Code on a dev laptop.

`rootUri` is a `file://` URI[0] pointing to the workspace root on the laptop.

`rootPath` (which was renamed to `workspaceRootPath` here) should be a filesystem path containing the workspace root on the server, which, prior to this change, was set to the first input directory passed to Sorbet.

This change will break Sorbet at Stripe, since it is setting `workspaceRootPath` to the filesystem path containing the workspace on the laptop.

[0] Note that it's not a `file://` URI for some other environments, like Monaco. I believe we have some automated tests for that use case.